src/App.tsx

```diff
@@ -1,61 +1,34 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
```

Comment:
[CODE KNOWLEDGE] - If you are using a non-mutating array method, you **don't** need to create a copy of the array. However, in this case, you are importing initial state for the todos which is fine as long as the imported value is not mutated directly.